### PR TITLE
Fix Clang 15/16 compatibility

### PR DESCRIPTION
**Pull request by thesamesam**

### CODE REVIEW

File: native/configure.ac [hunk 1]
```diff
@@ -303,9 +303,11 @@ AC_CHECK_HEADERS(sys/filio.h)
 AC_DEFUN([JK_CHECK_SETSOCKOPT], [
 AC_MSG_CHECKING(whether to use $1 with setsockopt())
 AC_TRY_RUN([
+#include <stdlib.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/time.h>
+#include <stdlib.h>
```
Comment (martin-g) [inline native/configure.ac:310]: No need of the second include. The one at line 306 should be enough.

Comment (thesamesam) [reply native/configure.ac:310]: Absolutely, I'll fix shortly.

**Review by martin-g — APPROVED** [review]

**Review by ChristopherSchultz — COMMENTED** [review]
Why is stdlib.h being added along with this commit?

**Comment by thesamesam** [comment]
> Why is stdlib.h being added along with this commit?

exit().

**Pull request merged**